server/multitenancy/tenant_resolver.ts

```diff
@@ -47,8 +47,8 @@ export function resolveTenant(
     selectedTenant = request.headers.securitytenant
       ? (request.headers.securitytenant as string)
```

Comment:
Just curious why both securitytenant and security_tenant are supported. Is this for supporting any legacy use case?

Reply:
yeah, that is implemented according to the existing code, which they need to support some legacy parameter names